pkgs/development/tools/misc/autoconf/2.71.nix

```diff
@@ -0,0 +1,69 @@
+{ lib, stdenv, fetchurl, m4, perl, texinfo }:
```

Comment:
Should we take this opportunity to add a common.nix file and share some of this code across versions?

Reply:
I personally find it slightly easier to reason about the packages when they don't reuse parts of their implementation. A few pros from the top of my head:
- deletion becomes just file (or directory) deletion
- package update touches a single version, makes it easier to reason about the impact for packages that are stdenv dependencies
- change history in `default.nix` is linear and does not get cluttered by occasional backports to older versions
- ofborg gets sad to assign reviewers until `pos` attributes (and extra pending fixes: https://github.com/NixOS/nixpkgs/pull/276892 ) are applied

It feels to me like `autoconf` does not contain much code to share and does not expect much churn in previous versions.

But if deduplication is a strong requirement I can try to reshuffle things a bit.

Reply:
I mean we can keep it like this, no strong feelings on my side.